reference/5-api-publisher/ConfigurationStore/Sql-Server.md

```diff
@@ -0,0 +1,233 @@
+# Configuration Store for SQL Server
+
+Enables management of individual connection settings with encryption support for securely storing keys and secrets using built-in support for symmetric key encryption in a SQL Server database.
+
+## Create the Ed-Fi API Publisher Configuration Database
+
+The following script template provides you with the information needed to create the `EdFi_API_Publisher_Configuration` database. Alternatively, you can perform this step manually using SSMS or through some other means and proceed with the next step.
+
+To provide parameters for the script, type <kbd>Ctrl</kbd>+<kbd>Shft</kbd>+<kbd>M</kbd> in the SSMS query window before executing.
+
+```sql
+
+-- IMPORTANT: Type Ctrl+Shft+M to enter parameters for the script.
+
+---------------------------------------------------------
+-- Create configuration database
+---------------------------------------------------------
+CREATE DATABASE [EdFi_API_Publisher_Configuration]
+ ON PRIMARY ( NAME = N'EdFi_API_Publisher_Configuration', FILENAME = N'< DataFilePath, nvarchar, C:\MSSQL\MSSQL14.MSSQLSERVER\MSSQL\DATA\EdFi_API_Publisher_Configuration.mdf >' )
+ LOG ON ( NAME = N'EdFi_API_Publisher_Configuration_log', FILENAME = N'< LogFilePath, nvarchar, C:\MSSQL\MSSQL14.MSSQLSERVER\MSSQL\DATA\EdFi_API_Publisher_Configuration_log.ldf >')
+GO
+```
+
+## Create the EdFiApiPublisher User
+
+The Ed-Fi ODS API Publisher is intended to use integrated security to connect to the configuration database. To that end, the process under which the publisher utility is executing should be a local or domain Windows identity with an associated SQL Server login named "EdFiOdsApiPublisher".
+
+To provide parameters for the script, type <kbd>Ctrl</kbd>+<kbd>Shft</kbd>+<kbd>M</kbd> in the SSMS query window before executing.
```

Comment:
Earlier I mentioned that we want to avoid HTML tags in our markdown, in general. This is actually a very nice use that we don't get with Markdown, so please go ahead and keep it!